README.md

```diff
@@ -1,21 +1,43 @@
 # ilab-on-ocp
 
-This repo will serve as the central location for the Containerfiles and yamls needed to deploy [Instructlab](https://instructlab.ai/) onto an OpenShift cluster with RHOAI.
+This repo will serve as the central location for the code, Containerfiles and yamls needed to deploy [Instructlab](https://instructlab.ai/) onto an [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift) cluster with [RHOAI](https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-ai). This project leverages and number of RHOAI tools working together to run InstructLab. Specifically, Data Science Pipelines, Kserve Serving, and the Distributed Training Operator.
 
-## Requirements
-The following Operators must be installed on the cluster
 
-* Red Hat - Authorino
-* NVIDIA GPU Operator
-* Node Feature Discovery
-* Red Hat OpenShift AI
-* Red Hat OpenShift Serverless
-* Red Hat OpenShift Service Mesh
+<p align="center"><img src="assets/images/completed_pipeline.png" width=50%\></p>
 
-### NVIDIA GPU Operator
-A ClusterPolicy must be deployed. The definition provided when clicking the "Create ClusterPolicy" although generic installs all required components.
 
-### Accelerator Profile
+## Getting Started
+
+This project makes running InstructLab based fine tuning easy and flexible on OpenShift. However, before you can get started there are a few prerequisites and additional setup steps that needs to be completed.
+
+### Cluster Requirements
+
+#### Operators:
+The following Operators must be installed on your OpenShift cluster:
+
+* [Red Hat OpenShift AI](need_link)
+* [Red Hat OpenShift Serverless](need_link)
+* [Red Hat OpenShift Service Mesh](need_link)
+* [Red Hat - Authorino](need_link)
+* [NVIDIA GPU Operator](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/getting-started.html)
+* [Node Feature Discovery](https://docs.openshift.com/container-platform/4.17/hardware_enablement/psap-node-feature-discovery-operator.html)
+
+#### Object Storage:
+
+Once the above operators have been successfully installed you will need to set up object storage for your models and pipeline artifacts. This solution requires object storage to be in place through [Noobaa](https://www.noobaa.io/).
```

Comment:
can be any S3 compatible storage, but we recommend nooba? and should we mention ODF?